vnext/Microsoft.ReactNative.Cxx/Crash.h

```diff
@@ -3,18 +3,22 @@
 
 #pragma once
 
+#ifndef VerifyElseCrash
 #define VerifyElseCrash(condition) \
   do {                             \
     if (!(condition)) {            \
       assert(false && #condition); \
       std::terminate();            \
```

Comment:
>terminate [](start = 11, length = 9)

Does this produce a crash report in Watson?

Reply:
I am not sure about it. We need to revisit this code.

---
In reply to: [377836562](https://github.com/microsoft/react-native-windows/pull/4076#discussion_r377836562) [](ancestors = 377836562)